tests/data/settings.py

```diff
@@ -26,3 +26,11 @@
 AUTOACL_FILE = os.environ.get('AUTOACL_FILE',
                               os.path.join(PREFIX, 'autoacl.py'))
 BOUNCE_FILE = os.environ.get('BOUNCE_FILE', os.path.join(PREFIX, 'bounce.py'))
+
```

Comment:
Once added to `trigger.conf.global_settings` you can safely remove this from here since it will then be a global default.